internal/store/files/recog_place_store.go

```diff
@@ -80,6 +82,25 @@ type RecogPlaceStore struct {
 	// recognized with their type preceding their name in the query. This is used
 	// to handle reconNames that are also common words like "to", "me", "a".
 	CommonWordReconNameToTypes map[string][]string
+	// US Zip Codes
+	USZipCodes map[string]struct{}
+}
+
+// RecogUSZipCode recognizes US zipcodes and returns a RecogPlaces object if it recognizes one.
+// If detected, the RecogPlaces includes a list of RecogPlace of len 1.
+// If not detected, it returns nil.
+func (store *RecogPlaceStore) RecogUSZipCode(token string) *pb.RecogPlaces {
+	if _, ok := store.USZipCodes[token]; !ok {
+		return nil
+	} else {
+		dcid := fmt.Sprintf("zip/%s", token)
+		fmt.Printf("%s, %s", token, dcid)
```

Comment:
did you mean to keep this print statement here?

Reply:
No - removed.